examples/gno.land/p/demo/grc/grc20/types.gno

```diff
@@ -12,9 +13,24 @@ var (
 	ErrInsufficientAllowance = errors.New("insufficient allowance")
 	ErrInvalidAddress        = errors.New("invalid address")
 	ErrCannotTransferToSelf  = errors.New("cannot send transfer to self")
+	ErrReadonly              = errors.New("banker is readonly")
+	ErrRestrictedBankOwner   = errors.New("restricted to bank owner")
 )
 
-type Token interface {
+type Bank struct {
```

Comment:
What about `Token`? I like it because if this is meant to be the "public object", then `var Foo grc20.Token` reads nicely. Or maybe `TokenInfo` (the "token" is not a data structure, but it is described by data structures).

Other idea, especially if we make this also compatible with the GRC20 interface readonly: make this a `Teller`, then the `account_banker.gno` functions `RealmTeller` and `VaultTeller` (ex `AccountBanker`). (See other comments.)